Tasks/UseDotNetV2/make.json

```diff
@@ -4,12 +4,27 @@
             "module": "../Common/packaging-common",
             "type": "node",
             "compile": true
+        },
+        {
+            "module": "../Common/utility-common-v2",
+            "type": "node",
+            "dest" : "./",
+            "compile" : true
         }
     ],
     "cp": [
         {
             "source": "externals",
             "options": "-R"
         }
+    ],
+    "rm": [
+        {
+            "items": [
+                "node_modules/utility-common-v2/node_modules/azure-pipelines-task-lib",
+				"node_modules/docker-common-v2/node_modules/azure-pipelines-task-lib"
```

Comment:
fix spacing. Do not use tabs. there is a setting in vs code which automatically converts tabs to spaces

Reply:
done